src/ZkFold/Cardano/OffChain/ECC.hs

```diff
@@ -0,0 +1,19 @@
+module ZkFold.Cardano.OffChain.ECC where
```

Comment:
Perhaps we should write a comment on how the abbreviation stands. ECC is Elliptic-curve cryptography